### PR TITLE
Fix rendering images

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

**Comment by j-f1** [comment]
That’s misleading — cancelled deploys show as successful

**Comment by j-f1** [comment]
I think this does it @vjeux

**Comment by vjeux** [comment]
It works! Thanks. Probably remove console.log :)

image

File: src/excalidraw-embed/get-image.js [hunk 1]
```diff
@@ -12,24 +12,22 @@ module.exports = async (url) => {
   const frame = await page.mainFrame();
   const result = await frame.evaluate(
     () =>
-      new Promise((resolve, error) => {
+      new Promise((resolve, rejec) => {
```
Comment (vjeux) [inline src/excalidraw-embed/get-image.js:15]: typo: `reject`

File: src/excalidraw-embed/get-image.js [hunk 2]
```diff
@@ -12,24 +12,22 @@ module.exports = async (url) => {
...
           button.click();
-        } catch (e) {
-          error(e);
+        } catch (error) {
+          reject(error);
         }
       })
   );
   await browser.close();
+  console.log("got result", result.slice(0, 100));
```
Comment (vjeux) [inline src/excalidraw-embed/get-image.js:31]: Remove

**Pull request merged**